src/components/Contentful/DynamicRichText.vue

```diff
@@ -23,12 +27,32 @@ export default {
 				},
 			};
 		},
+		elementClasses() {
+			let classes = 'dynamic-rich-text tw-whitespace-pre-wrap ';
+			if (this.bodyColumns) {
+				classes += 'tw-grid tw-gap-1 md:tw-gap-3 ';
+				if (this.bodyColumns?.md) {
+					classes += `md:tw-grid-cols-${this.bodyColumns.md} `;
+				}
+				if (this.bodyColumns?.lg) {
+					classes += `lg:tw-grid-cols-${this.bodyColumns.lg} `;
+				}
+				if (this.bodyColumns?.xl) {
+					classes += `xl:tw-grid-cols-${this.bodyColumns.xl} `;
+				}
+			} else {
+				classes += 'tw-prose';
```

Comment:
tw-prose sets spacing and type styles on various elements so it should probably be applied for all cases.

Reply:
tw-prose adds a a top and bottom margin to every element except for the first one. That lack of margin on the first element causes the grid items to be mis-aligned:
<img width="590" alt="Screen Shot 2022-09-15 at 9 50 37 AM" src="https://user-images.githubusercontent.com/4149025/190462926-1b820134-c9ac-4ba0-8916-c6cd62796f94.png">
<img width="627" alt="Screen Shot 2022-09-15 at 9 51 55 AM" src="https://user-images.githubusercontent.com/4149025/190462950-d9232adc-10b7-496c-9f21-1c634ab8eafe.png">

So to keep tw-prose those margins will have to be removed so that the spacing is handled by the grid instead when needed.

Reply:
oh yea, well that's a weird one... maybe we can address in tw-prose at somepoint. I was just thinking about when someone tries to use other elements or embeds with these blocks applied. Let's go with what you have for now.

How will the blocks be determined/configured in Contentful?

Reply:
Also, please review this code running locally against https://www.dev.kiva.org/lp/dynamic-hero-components + https://www.dev.kiva.org/lp/how-kiva-works to ensure the content layout of their dynamic hero usages are not altered.

Reply:
Ok so it's "activated" by adding configuration. That sounds good!

Thanks for verifying against the other pages!!!